.gitlab-ci.yml

```diff
@@ -105,8 +105,8 @@ test_enterprise-1.10:
     - source tmp/sdk-1.10/env.sh
     - tarantool -V
     - make ci_prepare
-    - make tmp/cache-image.tar
-    - docker load -i tmp/cache-image.tar
+    # - make tmp/cache-image.tar
```

Comment:
I suggest drop it or uncomment if your tests fail on the intermediate state it's ok

Reply:
Nothing fails, this code is redundant since there is no build in docker implemented. It just takes a lot of time in CI.

Reply:
Ok. Stay it as it is.